### DISCUSSION TITLE
How does Seurat handle sparsity in scRNAseq data

### DISCUSSION THREAD

**Discussion by jasminelmah** [satijalab/seurat#5011] [Q&A] [answered]
Recently, it's been suggested that zero-inflated models are not required for scRNAseq UMI data [(Valentine, 2020](https://www.nature.com/articles/s41587-019-0379-5); [Cao et al., 2021](https://www.nature.com/articles/s41587-020-00810-6). In other words, the zeros in UMI data are sufficiently described by non-zero inflated models, such as the Poisson or negative binomial. Does this mean that SCTransform, which is based on the negative binomial model, is sufficient to handle scRNAseq data sparsity? In a similar vein, given that something like LogNormalize does not explicitly model zeros, why is it still popular? Thanks for helping me understand!

**Comment by saketkc** [comment, marked as answer]
As you point out through the linked references, negative binomial is a sufficient (and [necessary](https://www.biorxiv.org/content/10.1101/2021.07.07.451498v1) )in explaining the number of zeros. LogNormalization is meant to achieve the same objective (1. correct for sequencing depth - scaling step and 2. reduce impact of outliers - log step) but can dampen biological variance at the same time (some slides [here](https://drive.google.com/file/d/1mbFNhQYHlVsKThIuhDfe_ouz-FkHdpGo/view)). The popularity comes probably because of ease of implementation (no explicit statistical estimation required).

**Reply by jasminelmah** [reply]
Thanks! That's very clarifying.
